core/src/test/java/com/sequenceiq/cloudbreak/service/publicendpoint/dns/BaseDnsEntryServiceTest.java

```diff
@@ -127,7 +129,7 @@ void testCreateOrUpdate() {
         when(dnsManagementService.createOrUpdateDnsEntryWithIp(USER_CRN, ACCOUNT_ID, FQDN_2, ENVIRONMENT_NAME, false, List.of(PUBLIC_IP_2))).thenReturn(true);
```

Comment:
I think would be wise to use any() instead of USER_CRN
IT can happen when  getUser() != USER_CRN

Reply:
or getting the actual getUser() and use it instead of USER_CRN

Reply:
Abandoning the usage of the USER_CRN field is not necessary since we're executing the method under test with this:

`ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.createOrUpdate(stack));`

so we're expecting the usage of this field on purpose